mlserver/settings.py

```diff
@@ -313,6 +316,12 @@ class ModelParameters(BaseSettings):
     """Path to the environment tarball which should be used to load this
     model."""
 
+    inference_pool_gid: Optional[str] = None
```

Comment:
I think that docs should follow this PR.

Reply:
I will do another PR to update the docs.